airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/DatabricksDestination.kt

```diff
@@ -97,8 +98,33 @@ class DatabricksDestination : BaseConnector(), Destination {
                 syncId = 0
             )
 
+        // quick utility method to drop the airbyte_check_test_table table
+        // returns a connection status if there was an error, or null on success
+        fun dropCheckTable(): AirbyteConnectionStatus? {
+            val dropCheckTableStatement =
+                "DROP TABLE IF EXISTS `${connectorConfig.database}`.`${streamId.rawNamespace}`.`${streamId.rawName}`;"
+            try {
+                destinationHandler.execute(
+                    Sql.of(
+                        dropCheckTableStatement,
+                    ),
+                )
+            } catch (e: Exception) {
+                log.error(e) { "Failed to execute query $dropCheckTableStatement" }
+                return AirbyteConnectionStatus()
+                    .withStatus(AirbyteConnectionStatus.Status.FAILED)
+                    .withMessage("Failed to execute $dropCheckTableStatement: ${e.message}")
```

Comment:
I don't think we behave consistently across destinations wrt how we expose the actual error message here. Probably a thing we should think about for the new cdk